anitya/lib/backends/rubygems.py

```diff
@@ -74,3 +74,27 @@ def get_versions(cls, project):
                 'Project or version unknown at %s' % url)
 
         return [data['version']]
+
+    @classmethod
+    def check_feed(cls):
+        ''' Return a generator over the latest 50 uploads to rubygems.org
+
```

Comment:
@ralphbean started to answer it at: https://github.com/fedora-infra/anitya/pull/300#discussion_r62793372

But RFE upstream would be cool nonetheless as it would reduce the amount of data we ask them.